pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[build-system]
+requires = ["setuptools>=62.6", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dask-image"
+authors = [{name="dask-image contributors. see https://github.com/dask/dask-image/graphs/contributors"}]  # noqa: E501
+description = "Distributed image processing"
+keywords = ["dask-image", "dask", "image"]
+readme = "README.rst"
+license = { file="LICENSE.txt" }
+dynamic = ["version"]
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
```

Comment:
Maybe worth revisiting if this is still accurate? Perhaps that can be a separate discussion though

Reply:
Yes, maybe should open an issue to discuss.

Reply:
Issue opened here: https://github.com/dask/dask-image/issues/371